SWLOR.Game.Server/Service/CraftService.cs

```diff
@@ -233,13 +233,14 @@ public static float CalculateCraftingDelay(NWPlayer oPC, int skillID)
             SkillType skillType = (SkillType)skillID;
 
             // Identify which perk to use for this skill.
-            if (skillType == SkillType.Weaponsmith) perkType = PerkType.SpeedyWeaponsmith;
-            else if (skillType == SkillType.Armorsmith) perkType = PerkType.SpeedyArmorsmith;
-            else if (skillType == SkillType.Cooking) perkType = PerkType.SpeedyCooking;
-            else if (skillType == SkillType.Engineering) perkType = PerkType.SpeedyEngineering;
-            else if (skillType == SkillType.Fabrication) perkType = PerkType.SpeedyFabrication;
-            else if (skillType == SkillType.Medicine) perkType = PerkType.SpeedyMedicine;
-            else if (skillType == SkillType.Harvesting) perkType = PerkType.SpeedyReassembly;
+            if (skillType == SkillType.Weaponsmith) perkType = PerkType.SpeedyCrafting;
```

Comment:
You're killing me, smalls! Drop all these unnecessary if statements broski. Just a single line for perkType (you could even get rid of the variable, but I'd settle for just simplifying the if statements)